src/store/flow/utils/importHelpers.ts

```diff
@@ -49,6 +49,64 @@ export function mergeFlowContainer(
   return existingFlowContainerStore
 }
 
+interface CreateFlowStackBlock {
+  uuid: string;
+  type: string;
+  config: {
+    flow_id: string;
+  };
+}
+
+interface CreateFlowStackFlow {
+  uuid: string;
+  blocks: CreateFlowStackBlock[];
+}
+
+interface CreateFlowStackData {
+  flows: CreateFlowStackFlow[];
+}
+
+export function createContainerFlowStack(json_data: any, flow_stack: string[]): void {
```

Comment:
This method is quite complex to understand. I'd recommend we create an unit testing to cover this, in case we make some changes later, the test cases will confirm we get it right

- [ ] add unit testing for this

Reply:
@lakhbawa could you also add a JSDoc to this function explaning how it works and behaves?

Reply:
In my understanding, you're creating a hierarchy of flows - how they use each other via RunFlowBlock. But the details of this hierarchy are unclear.

What if a flow has multiple RunFlowBlocks, and they have their own children, etc.
How would they be ordered in the stack?
Will all of the flows in the container be included in the stack?
Or only one 'lineage' of flows from top to botton, ignoring some siblings?

Reply:
fyi: we have test sample here https://github.com/FLOIP/flow-builder/blob/master/tests/unit/components/interaction-designer/block-editors/choices/expressionTransformers.spec.ts

Reply:
Thanks @bulatgab @safydy-r added description + test